compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -405,6 +405,8 @@ void addVMVXDefaultPassPipeline(OpPassManager &passManager,
   addTileAndDistributePasses(passManager);
 
   if (enableMicrokernels) {
+    passManager.nest<ModuleOp>().nest<func::FuncOp>().addPass(
+        createDecomposeBatchMmt4DOpsPass());
```

Comment:
I'd prefer adding this with an e2e test. It can be a separate PR. For this PR, let's focus on `Add support for tiling batch dimension with size=1 in batch_mmt4d decomposition`.

Reply:
Sure, will raise a separate patch for the this.